pkg/graph/schema/types/possible_operational_solution_contact.graphql

```diff
@@ -9,6 +9,7 @@ type PossibleOperationalSolutionContact {
   email: String!
   isTeam: Boolean!
   role: String
+  isPrimary: Boolean

```

Comment:
Per my previous comment (in the migration), I think we can make this not nullable

```suggestion
  isPrimary: Boolean!
```